packages/sign_in_with_apple/lib/src/sign_in_with_apple.dart

```diff
@@ -197,7 +197,7 @@ class SignInWithApple {
             .join(' '),
         // Request `code`, which is also what `ASAuthorizationAppleIDCredential.authorizationCode` contains.
         // So the same handling can be used for Apple and 3rd party platforms
-        'response_type': 'code',
+        'response_type': 'code+id_token',
```

Comment:
@tp Do you think it makes sense to configure this from the outside?

Reply:
FYI: I added the `id_token` response param according to https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_js/incorporating_sign_in_with_apple_into_other_platforms#3332113

The `code+id_token` has been taken from https://forums.developer.apple.com/thread/118209. I still need to validate this

Reply:
Update: I tried the android code and it seems to be `code id_token` in the query param so I adjusted the code for this.

Reply:
👍